Runtime/Scripts/Settings/Data/MeshSyncPlayerConfig.cs

```diff
@@ -102,6 +103,8 @@ internal abstract class MeshSyncPlayerConfig : ISerializationCallbackReceiver {
     //Sync Settings
     public bool SyncVisibility = true;
     public bool SyncTransform  = true; //for Update
+
+    public bool SyncInstances = false; 
```

Comment:
This should be in `MeshSyncServerConfig`, unless we need to support this for SceneCache as well